IntroScenes/FirstStoryCutScene.gd

```diff
@@ -31,226 +31,119 @@ func _ready():
 #func _process(delta):
 #	pass
 func d1():
-	newdialog([
-	"10 years ago...",
-	TrainerName + "'s family lived in Bealbeach City,",
-	"the largest city in Tandor."
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D1"), true)
 	dialog.rect_position = MIDDLE
 	pass
 	
 func d2():
-	newdialog([
-	"Their father, Kellyn, was a brave",
-	"and talented Pokémon Ranger."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D2"), false)
 	dialog.rect_position = BOTTOM
 	pass
 func d3():
-	newdialog([
-	"Their mother, Lucille, was a brilliant",
-	"scientist and an expert on Nuclear Energy."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D3"), false)
```

Comment:
I think you missed some of the `newdialog`. Should be `new_dialog`

Reply:
Fixed